src/map/lua/lua_baseentity.cpp

```diff
@@ -14505,7 +14505,7 @@ void CLuaBaseEntity::Register()
     SOL_REGISTER("setAggressive", CLuaBaseEntity::setAggressive);
     SOL_REGISTER("setTrueDetection", CLuaBaseEntity::setTrueDetection);
     SOL_REGISTER("setUnkillable", CLuaBaseEntity::setUnkillable);
-    SOL_REGISTER("untargetable", CLuaBaseEntity::untargetable);
+    SOL_REGISTER("setUntargetable", CLuaBaseEntity::setUntargetable);
```

Comment:
Though unused, should we go ahead and create a getUntargetable lua binding to match?

Reply:
I did think about that, will do

Reply:
Yes, I'm fully updated and of course, compiled thereafter.